### PR TITLE
Configurable email addresses for Alert emails

### PR DESCRIPTION
**Pull request by majikandy**
Configure using the variables in the `.env`

```
TF_VAR_SUPPORT_EMAIL_ADDRESSES='["email1@hotmail.com","email2@hotmail"]'
```

### Acceptance Criteria

- [ ] alerts go to the first email configured
- [ ] alerts go to the second email configured
- [ ] this works on dev / non-prod / prod

### Testing information

Alerts can be viewed here - https://portal.azure.com/#blade/Microsoft_Azure_Monitoring/AlertsManagementSummaryBlade

But they will also go to the configured email addresses.

### Pre-flight Checklist

- [ ] Testing aligns with [testing strategy][testing strategy]
- [ ] DUXD review approval
- [ ] Accessibility Reviewed
- [ ] Product owner demo

### Pre-merge Checklist

- [ ] Branch test approved

[testing strategy]: https://github.com/MHRA/products/blob/master/docs/principles/testing.md "MHRA/products testing strategy"

### CODE REVIEW

File: infrastructure/environments/dev/.env.example [hunk 1]
```diff
@@ -3,3 +3,6 @@ TF_VAR_CLIENT_SECRET=
 
 TF_VAR_CLUSTER_ROUTE_DESTINATION_CIDR_BLOCKS='["10.0.0.0/16"]'
 TF_VAR_CLUSTER_ROUTE_NEXT_HOP=
+
+TF_VAR_SUPPORT_EMAIL_ONE=
+TF_VAR_SUPPORT_EMAIL_TWO=
```
Comment (sgrowe) [inline infrastructure/environments/dev/.env.example:8]: It's probably a good idea to just have a single array of email addresses similar to how `TF_VAR_CLUSTER_ROUTE_DESTINATION_CIDR_BLOCKS` works above

Comment (majikandy) [reply infrastructure/environments/dev/.env.example:8]: it need to produce multiple blocks in the terraform config, will that be easy ?

Comment (majikandy) [reply infrastructure/environments/dev/.env.example:8]: yes please to pairing @sgrowe

Comment (sgrowe) [reply infrastructure/environments/dev/.env.example:8]: It's shouldn't too hard, I think this something like this code should do the trick:

```tf
resource "azurerm_monitor_action_group" "support" {
  name                = "CriticalAlertsAction"
  resource_group_name = var.resource_group_name
  short_name          = "support"

  dynamic "email_receiver" {
    for_each = var.support_emails

    content {
      name                    = email_receiver.value
      email_address           = email_receiver.value
      use_common_alert_schema = true
    }
  }
}
```

File: infrastructure/environments/dev/.env.example [hunk 2]
```diff
@@ -3,3 +3,5 @@ TF_VAR_CLIENT_SECRET=
 
 TF_VAR_CLUSTER_ROUTE_DESTINATION_CIDR_BLOCKS='["10.0.0.0/16"]'
 TF_VAR_CLUSTER_ROUTE_NEXT_HOP=
+
+SUPPORT_EMAIL_ADDRESSES=
```
Comment (TimboTambo) [inline infrastructure/environments/dev/.env.example:7]: Think this should be `TF_` prefixed

Comment (majikandy) [reply infrastructure/environments/dev/.env.example]: good spot

**Pull request merged**